omnibus/config/software/more-ruby-cleanup.rb

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright:: 2019 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "fileutils"
+
+name "more-ruby-cleanup"
+
+skip_transitive_dependency_licensing true
+license :project_license
+
+source path: "#{project.files_path}/#{name}"
+
+dependency "ruby"
+dependency "rubygems"
+
+build do
+  block "Removing additional non-code files from installed gems" do
+    # find the embedded ruby gems dir and clean it up for globbing
+    target_dir = "#{install_dir}/embedded/lib/ruby/gems/*/gems".tr('\\', "/")
+    files = %w{
+      .github
+      examples
+      example
+      docs
+      doc
+      doc-api
+      sample
+      samples
+      test
+      tests
+      benchmark
+      benchmarks
+      rakelib
+      CHANGELOG.md
+      CONTRIBUTORS.md
+      README.md
+      README.markdown
+      HISTORY.md
+      TODO.md
+      CONTRIBUTING.md
+      ISSUE_TEMPLATE.md
+      UPGRADING.md
+      CODE_OF_CONDUCT.md
+      Code-of-Conduct.md
+      ARCHITECTURE.md
+      CHANGES.md
+      README.YARD.md
+      GUIDE.md
+      MIGRATING.md
+      README.txt
+      HISTORY.txt
+      Manifest.txt
+      Manifest
+      CHANGES.txt
+      CHANGELOG.txt
+      warning.txt
+      FAQ.txt
+      release-script.txt
+      TODO
+      HISTORY
+      CHANGES
+      CHANGELOG
+      README
+      Gemfile
```

Comment:
think you broke appbundler here

Reply:
This is at the end once appbundler has already done its thing. Is that going to be a problem still?